modules/microshift-get-cluster-id-kubesystem.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// microshift_support/microshift-getting-cluster-id.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-get-cluster-id-kubesystem_{context}"]
+= Getting your cluster ID using kube-system namespace
+
+.Procedure
+
+* Get the namespace UID of the running cluster by entering the following command:
++
+[source,terminal]
+----
+$ oc get namespaces kube-system -o jsonpath={.metadata.uid}
+----
+
+* Get the namespace UID of a non-running cluster by entering the following command:
++
+[source,terminal]
+----
+$ sudo cat /var/lib/microshift/cluster-id
+----
+
+[NOTE]
+====
+The {microshift-short} service copies the `kube-system` namespace metadata UID to the `cluster-id` file so that the UID can be accessed when the cluster is not running. The `cluster-id` file is found in the `/var/lib/microshift` directory.
+====
+
+* You can view the {microshift-short} database files in the directory '/var/lib/microshift' by entering the following command.
++
+[source,terminal]
+----
+$ sudo ls -l /var/lib/microshift
+----
+.Example output of {microshift-short} database file in the directory '/var/lib/microshift'.
++
+[source,terminal]
+----
+total 16
+drwxr-xr-x. 14 root root 4096 Jan 28 12:03 certs
+-r--------.  1 root root   36 Jan 28 12:04 cluster-id
+drwx------.  3 root root   20 Jan 28 12:03 etcd
+drwxr-xr-x.  3 root root   20 Jan 28 12:04 kubelet-plugins
+drwxr-xr-x. 10 root root 4096 Jan 28 12:03 resources
+-rw-------.  1 root root   65 Jan 28 12:03 version
+----
```

Comment:
```suggestion
* You can view the {microshift-short} database files in the directory '/var/lib/microshift' by entering the following command.
+
[source,terminal]
----
$ sudo ls -l /var/lib/microshift
----
.Example output of {microshift-short} database file in the directory '/var/lib/microshift'.
+
[source,terminal]
----
total 16
drwxr-xr-x. 14 root root 4096 Jan 28 12:03 certs
-r--------.  1 root root   36 Jan 28 12:04 cluster-id
drwx------.  3 root root   20 Jan 28 12:03 etcd
drwxr-xr-x.  3 root root   20 Jan 28 12:04 kubelet-plugins
drwxr-xr-x. 10 root root 4096 Jan 28 12:03 resources
-rw-------.  1 root root   65 Jan 28 12:03 version
----
```
This should be a part of the [NOTE], because the section is about getting UID which is being done by first two steps. Please check if you can include this part in the note.

Reply:
I think this should not be part of the note. 
@ShaunaDiaz  what do you suggest?

Reply:
@aravipra No, this should not go in the Note. Question: what is the output of Step 1? Is that pulling the cluster ID straight from the running cluster and not from the directory? If so, there might need to be two modules, one for getting the ID from a running cluster and one from a non-running cluster.

Reply:
as per our conversation I am making changes to the modules